### DISCUSSION TITLE
Will hasura price change effect us as well?

### DISCUSSION THREAD

**Discussion by ankur1163** [nhost/nhost#1579] [General]
hasura has hiked their prices. It will effect nhost as well? 
Thanks in advance

**Comment by elitan** [comment]
No, we're using the open source version so Hasura Cloud's pricing has no effect on Nhost.

**Comment by ammarfaris** [comment]
Do you think more open-source Community Edition features will get reduced/axed/depreciated over time? Like the case when they depreciated support (starting Feb 2023) for databases that are primarily designed for enterprise use cases, including: Google BigQuery, Amazon Athena, Snowflake, and Microsoft SQL Server, as well as advanced PostgreSQL connectors such as Azure Cosmos DB/Citus and CockroachDB. ( https://hasura.io/blog/updates-to-hasura-pricing-and-plans/ )

Does Nhost has future plans to implement crucial features that are missing from the CE version, such as caching and rate-limiting? 

Thanks for being awesome, Nhost!

@elitan

**Reply by dbarrosop** [reply]
> Do you think more open-source Community Edition features will get reduced/axed/depreciated over time?

Anything we say is going to be speculation but, in any case, I'd expect the core functionality to remain the same.

> Does Nhost has future plans to implement crucial features that are missing from the CE version, such as caching and rate-limiting?

It's not in the roadmap as of now. For people needing caching and/or security features we recommend using [stellate](https://stellate.co). In the future we may want to integrate hasura directly with them but right now users can do it themselves or we can do it as part of an enterprise plan.

**Reply by ammarfaris** [reply]
Thanks for your swift response @dbarrosop. I'll take a look at Stellate.

**Reply by ammarfaris** [reply]
@dbarrosop would like to ask for some pointers on how to setup stellate **so that it will work with nhost's auth and storage as well**  - because stellate link/paths will only process Graphql requests; when having stellate link as my sole Nhost client backendUrl,  I received this message when trying to sign in (i.e when trying to access www.my-stellatelink.com/v1/auth/signin/provider/google)

```{"errors":[{"message":"404: Not found. Only \"/\", \"/graphql\", and \"/log\" is allowed."}]}```

so far I tried to create two nhost clients,  with the first one for auth and storage purpose while the second one for Graphql requests purpose ; but it does not seem to work with that setup....

```
<NhostProvider nhost={withOriginalLink}>
    <NhostApolloProvider nhost={withStellateLink}>
```

Appreciate some pointers here... Thanks in advance for your help.. Kudos to the great work the Nhost team is doing!

**Reply by dbarrosop** [reply]
If you are using our cloud service storage already includes a fully integrated CDN with instant purging so you shouldn't need to add another layer. I don't think auth is a good fit for caching as it is dealing with sessions and other data that may expire quite quickly (it is also lightweight enough that isn't probably worth it either).
